packages/okta-vue/yarn.lock

```diff
@@ -16,6 +16,10 @@
     esutils "^2.0.2"
     js-tokens "^4.0.0"
 
+"@okta/configuration-validation@^0.3.0":
```

Comment:
Why are these updating to 0.3.0 here (where you're creating 0.4.0)?

Reply:
I have left the versions untouched in the various packages. The first one to require 0.4 will be React in a subsequent PR. I can update everything to 0.4 if we prefer.

Reply:
I'm more curious why your changes to configuration-validation are making a change in the other packages at all

Reply:
For some reason their reference to configuration-validation was missing in the yarn.lock. I noticed it because it failed on `--frozen-lockfile`. It's not entirely clear to me at this moment how it was working beforehand (with this entry missing in the lock)

Reply:
Ah.  I'm not bothered by this being 0.3.0 then